### ISSUE TITLE
pretrained steganalysis model weights.

### ISSUE THREAD

**Issue by seoyunYang** [cxy1997/LISO#3]
Hello, may I ask what datasets and steganography models were used to train siastegnet and xunet? 
Thank you!

**Comment by cxy1997** [comment]
The steganalysis models are trained on CelebA's validation set. 

The steganography images used in training are produced by LISO 1-4 bit models that are trained on CelebA's training set.

**Comment by seoyunYang** [comment]
Thank you!!
